src/app/store/Config/Config.action.js

```diff
@@ -9,8 +9,14 @@
  * @link https://github.com/scandipwa/base-theme
  */
 export const UPDATE_CONFIG = 'UPDATE_CONFIG';
+export const SET_LOADING_STATUS = 'SET_LOADING_STATUS';
```

Comment:
How is that needed? Why? Just either show blank or logo if it is loaded.

Reply:
According to the ticket, 
> Default logo - ScandiPWA. If there is option to delete logo, then our logo will show as a placeholder.
Should I make it blank anyway?